web/components/cycles/archived-cycles/header.tsx

```diff
@@ -61,6 +62,13 @@ export const ArchivedCyclesHeader: FC = observer(() => {
     }
   };
 
+  const appliedFilters: TCycleFilters = {};
```

Comment:
Since we are not using applied filters for display purposes and only dependent on the boolean value, can you bailout early as soon as you come across a filter.

```
const hasFilters = objectHasValues(currentProjectArchivedFilters ?? {})

// Put this in a utilities file
objectHasValues = (obj)=>{
	// run the for loop here.
}
```